<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetricsDialog</class>
 <widget class="QDialog" name="MetricsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelWaiterId">
      <property name="text">
       <string>ID kelnera:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="waiterIdSpinBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnLoadMetrics">
      <property name="text">
       <string>Poka≈º metryki</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTableWidget" name="tableMetrics">
      <property name="columnCount">
       <number>2</number>
      </property>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
